java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/test/ErrataHandlerTest.java

```diff
@@ -658,6 +658,14 @@ public void testRemovePackages() throws Exception {
     }
 
     public void testCloneErrata() throws Exception {
+        cloneErrataTest(false);
+    }
+
+    public void testCloneVendorErrata() throws Exception {
+        cloneErrataTest(true);
+    }
+
+    public void cloneErrataTest(boolean vendor) throws Exception {
```

Comment:
Why is this method public?

Reply:
That's a mistake. I created the method by changing its name from a public one. Will fix.